src/components/shared/select.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+export class Select extends React.Component {
```

Comment:
Why is this a class component? Why not make it a functional one? What's the need behind making it a class?

Reply:
Fixed select is a functional component